proj-sys/src/lib.rs

```diff
@@ -28,6 +28,11 @@
 //! implement your own set of callbacks if you wish to make use of them (see the
 //! [`proj`](https://crates.io/crates/proj) crate for an example).
 
+#[cfg(bundled_build)]
+extern crate libsqlite3_sys;
```

Comment:
Why do we need these `extern crate` declarations?

Reply:
Rustc only links crates that are explictly used somewhere, either by actually use items from that crate or by having an explicit `extern crate …` statement somewhere. So this essentially ensures that we actually link `libsqlite3` (the c library) for the bundled build case. We set the cfg flag from the build script if we actually build libproj from source and statically link it. Only then we need to also link libsqlite3.